index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <div id="img-div">
```

Comment:
I think it would be better to use semantic tags for the Header instead of a div

Reply:
- [ ] You can add emojis to your comments to make it more friendly, kindly